web_remember_tree_column_width/static/src/js/list_renderer.esm.js

```diff
@@ -0,0 +1,62 @@
+/** @odoo-module **/
+
+import {ListRenderer} from "@web/views/list/list_renderer";
+import {browser} from "@web/core/browser/browser";
+import {patch} from "web.utils";
+
+patch(ListRenderer.prototype, "web_remember_tree_column_width.ListRenderer", {
+    /**
+     * @override
+     */
+    computeColumnWidthsFromContent() {
+        const columnWidths = this._super.apply(this, arguments);
+        const table = this.tableRef.el;
+        const thElements = [...table.querySelectorAll("thead th:not(.o_list_button)")];
+        const self = this;
+        thElements.forEach(function (el, elIndex) {
+            const fieldName = $(el).data("name");
+            if (self.props.list.resModel && fieldName && browser.localStorage) {
+                const storedWidth = browser.localStorage.getItem(
+                    "odoo.columnWidth." + self.props.list.resModel + "." + fieldName
```

Comment:
For this line, we can use `template literals` here for more simple

![image](https://github.com/OCA/web/assets/62236456/a2114459-924e-4405-b8fe-f81cd8917c34)

Source: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#syntax